po/it.po

```diff
@@ -1,16 +1,16 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
-"POT-Creation-Date: 2023-09-19\n"
-"PO-Revision-Date: 2023-09-19 09:35+0100\n"
+"POT-Creation-Date: 2023-10-19\n"
```

Comment:
Hi @detro, updating this by hand will work, but it's not the intended workflow (I can tell that you did it by hand since it has only a date without a time).

You need to run `cargo install mdbook-i18n-helpers` to get version 0.2.4 of our helper tools. This will ensure that your `messages.pot` file gets a POT-Creation-Date field set to now, including hours, minutes, and seconds.

When you run `msgmerge`, this field is copied to the `it.po` file. This way we know when the English text was last extracted and we use this when publishing the course:

https://github.com/google/comprehensive-rust/blob/9637e3c6079c00f512f54bd923e8f71e37fd3e4a/.github/workflows/publish.yml#L52-L58

Reply:
Mmm, I knew that what I did I wasn't supposed to do.

I did look at your previous CL about this, and probably misunderstood what I was supposed to do.

Let me fix it.

Reply:
That's very fair! I was the one who started by "cheating" here :smile:

When I introduced the new "snapshot publishing" system, I needed to add a POT-Creation-Date field to each file. This ensure that the PO files stop getting more out of date. So I just used the date of the last edit as a first approximation — knowing that it the field would be made correct the next time someone creates a `messages.pot` file and runs `msgmerge`.